go/ql/lib/change-notes/released/0.7.3.md

```diff
@@ -2,7 +2,7 @@
 
 ### Minor Analysis Improvements
 
-* Added the [gin cors](https://github.com/gin-contrib/cors) library to the CorsMisconfiguration.ql query
+* Added the [gin cors](https://github.com/gin-contrib/cors) library to the experimental query "CORS misconfiguration" (`go/cors-misconfiguration`).
```

Comment:
Should CORS be capitalised in "gin CORS" while we are at it? (Perhaps that would also be better worded as "CORS for gin" or similar.)